ee/kafka_client/test/test_client.py

```diff
@@ -28,3 +31,42 @@ def test_kafka_produce_and_consume(self):
         producer.produce(topic=self.topic, data=self.payload)
         payload = next(consumer)
         self.assertEqual(payload.value, self.payload)
+
+    def test_kafka_default_security_protocol(self):
+        producer = _KafkaProducer(test=False)
+        self.assertEqual(producer.producer.config["security_protocol"], "PLAINTEXT")
```

Comment:
Looks like this test is failing, from the [test run](https://github.com/PostHog/posthog/runs/6006002876):
```
ee/kafka_client/test/test_client.py:37: error: "TestKafkaProducer" has no attribute "config"
ee/kafka_client/test/test_client.py:56: error: "TestKafkaProducer" has no attribute "config"
ee/kafka_client/test/test_client.py:72: error: "TestKafkaProducer" has no attribute "config"
```